packages/sfml/sfml.0.09.2/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer: "Florent Monnier <monnier.florent@gmail.com>"
+authors: [
+  "Florent Monnier"
+  "Sylvain Boilard"
+]
+
+homepage: "https://github.com/fccm/ocaml-sfml"
+dev-repo: "git+https://github.com/fccm/ocaml-sfml.git"
+bug-reports: "https://github.com/fccm/ocaml-sfml/issues"
+doc: "http://fccm.github.io/ocaml-sfml/"
+
+license: "zlib/png"
```

Comment:
Does this stand for https://spdx.org/licenses/zlib-acknowledgement.html ?
In that case
```suggestion
license: "zlib-acknowledgement"
```
```

Reply:
quoting [sfml@github](https://github.com/SFML/SFML):
The SFML libraries and source code are distributed under the [zlib/libpng license](https://opensource.org/licenses/Zlib).